helm/hpcc/templates/_helpers.tpl

```diff
@@ -860,13 +860,19 @@ resources:
 
 {{/*
 Add resources object for stub pods
-Pass in dict with instances defined
-*/}}
-{{- define "hpcc.addStubResources" }}
+Pass in dict with root, me and instances defined
+*/}}
+{{- define "hpcc.addStubResources" -}}
+{{- $stubInstanceResources := .root.Values.global.stubInstanceResources | default dict -}}
+{{- $milliCPUPerInstance := $stubInstanceResources.cpu | default "50m" -}}
+{{- $memPerInstance := $stubInstanceResources.memory | default "200Mi" -}}
+{{- $milliCPUs := int (include "hpcc.k8sCPUStringToMilliCPU" (dict "value" $milliCPUPerInstance)) -}}
+{{- $bytes := int64 (include "hpcc.k8sMemoryStringToBytes" (dict "value" $memPerInstance)) -}}
+{{- $totalBytes := mul .instances $bytes }}
 resources:
   limits:
-    cpu: "50m"
-    memory: {{ (printf "%dM" (mul .instances 100)) | quote }}
+    cpu: {{ printf "%dm" (mul .instances $milliCPUs) | quote }}
+    memory: {{ include "hpcc.bytesToK8sMemoryString" (dict "value" $totalBytes) | quote }}
```

Comment:
If you are just passing a single value, there's not really a need to use a dict and pass it by name...

Reply:
I didn't know you could, thought had to be a scope, which meant a dictionary..